### PR TITLE
[MIG] website_sale_product_assortment: Migration to 14.0

### PR DESCRIPTION
**Pull request by AshishHirapara**
Module migration to 14.0

### CODE REVIEW

File: website_sale_product_assortment/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{
+    "name": "eCommerce product assortment",
+    "summary": "Use product assortments to display products available on e-commerce.",
+    "version": "14.0.1.0.0",
+    "development_status": "Beta",
+    "license": "AGPL-3",
+    "category": "Website",
+    "website": "https://github.com/OCA/e-commerce",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "maintainers": ["CarlosRoca13", "ashishhirpara"],
```
Comment (CarlosRoca13) [inline website_sale_product_assortment/__manifest__.py]:
```suggestion
    "maintainers": ["CarlosRoca13"],
```

**Review by CarlosRoca13 — CHANGES_REQUESTED** [review]
To be a module maintainer you could at least make some improvements. But changing the version number doesn't give you the right to be a maintainer.

**Review by francesco-ooops — APPROVED** [review]
functional review ok!

File: website_sale_product_assortment/__manifest__.py (hunk 1 above)
Comment (AshishHirapara) [reply website_sale_product_assortment/__manifest__.py]: ok, done.

**Review by elvise — APPROVED** [review]
LGTM!

**Review by CarlosRoca13 — APPROVED** [review]

File: website_sale_product_assortment/__manifest__.py (hunk 1 above)
Comment (CarlosRoca13) [reply website_sale_product_assortment/__manifest__.py]: Thanks

**Pull request merged**